scripts/nayduck.py

```diff
@@ -203,6 +207,24 @@ def main():
         'tests': list(tests)
     }
 
+    if args.run_locally:
+        for test in post["tests"]:
+            # TODO: use parses similar to nayduck, where is it?
+            fields = test.split(" ")
```

Comment:
`fields = test.split()` without an argument so that it works with consecutive white space and with any white space characters (not just space).